### PR TITLE
Consistently apply `.editorconfig` defined rules across whole codebase

### PR DESCRIPTION
**Pull request by vladimyr**
Every time someone makes code changes there is risk of getting _bloated_ diffs due to VSCode (or any other EditorConfig aware editor) applying code style rules automatically.

For example line endings configuration is following:
https://github.com/vladimyr/dmn/blob/2bb1a7cc7e7e7a8e9d64ceaeca97131a14e3c08d/.editorconfig#L7

and current state of codebase:
```
➜  dmn git:(master) file **/* 
bin:                         directory 
bin/dmn:                     Node.js script, ASCII text executable
CHANGELOG:                   ASCII text
index.js:                    ASCII text, with CRLF line terminators
lib:                         directory 
lib/clean.js:                ASCII text
lib/cli.js:                  ASCII text
lib/console_ex.js:           ASCII text, with CRLF line terminators
lib/gen.js:                  ASCII text
lib/targets.js:              ASCII text
LICENSE:                     ASCII text, with CRLF line terminators
package.json:                ASCII text
package-lock.json:           ASCII text
README.md:                   ASCII text, with very long lines, with CRLF line terminators
test:                        directory 
test/fixtures:               directory 
test/fixtures/clean.test.js: ASCII text, with CRLF line terminators
test/fixtures/gen.test.js:   ASCII text
test/run_tests.js:           ASCII text
tools:                       directory 
tools/add_targets.js:        ASCII text
```

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -11,5 +11,5 @@ trim_trailing_whitespace = true
 indent_style = space
 indent_size = 4
 
-[{package.json,.travis.yml}]
-indent_size = 2
\ No newline at end of file
+[{package.json,package-lock.json,travis.yml}]
```
Comment (silverwind) [inline .editorconfig]: missing `.`

Comment (vladimyr) [reply .editorconfig]: My bad 🤦

**Comment by Trott** [comment]
This needs a rebase. Can you take care of that?

**Comment by vladimyr** [comment]
@Trott Done

**Pull request merged**